### PR TITLE
Doc updates and release notes for 3.2.12-3.2

### PR DESCRIPTION
**Pull request by azhebel**

### CODE REVIEW

File: source/release_notes/3.2.12-3.2.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+* :jirabug:`PSMDB-17`: Changed welcome message in the shell
+  to mention |PSMDB| instead of *MongoDB*
+
+* :jirabug:`PSMDB-90`: Added error message for storage engines
+  that do not support :ref:`hot-backup`
+
+* :jirabug:`PSMDB-91`: Deprecated ``audit`` configuration section
+  and added ``auditLog`` instead
+
+* :jirabug:`PSMDB-94`: Fixed ``audit_create_database`` test
```
Comment (denis-protivenskii) [inline source/release_notes/3.2.12-3.2.rst]: I don't think someone is interested in what tests we fixed, it should not be in the release notes. Please, remove all the tickets mentioning tests.

File: source/release_notes/3.2.12-3.2.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+* :jirabug:`PSMDB-95`: Fixed version dependencies for subpackages
+  so that all corresponding packages get updated accordingly
+
+* :jirabug:`PSMDB-96`: Excluded :file:`diagnostic.data` directory
```
Comment (denis-protivenskii) [inline source/release_notes/3.2.12-3.2.rst:40]: It's for PerconaFT backup only.

File: source/release_notes/3.2.12-3.2.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+* :jirabug:`PSMDB-101`: Implemented the ``auditAuthorizationSuccess``
+  parameter to :ref:`enable auditing of authorization success <auditAuthorizationSuccess>`
+
+* :jirabug:`PSMDB-104`: Updated links in client shell output
+  to point to |PSMDB| documentation and forum
+
+* :jirabug:`PSMDB-107`: Fixed behavior when creating the
+  :ref:`audit log file <audit-log>`
+
+* :jirabug:`PSMDB-111`: Refactored ``external_auth`` tests
```
Comment (denis-protivenskii) [inline source/release_notes/3.2.12-3.2.rst]: Ditto (tests).

File: source/release_notes/3.2.12-3.2.rst (hunk 1 above)
Comment (hrvojem) [reply source/release_notes/3.2.12-3.2.rst]: For Percona Server we use: "Other bugs fixed" for this.

**Pull request merged**